src/openMVG/matching_image_collection/Cascade_Hashing_Matcher_Regions_AllInMemory.cpp

```diff
@@ -199,30 +200,31 @@ void Match
         ++my_progress_bar;
         if (!vec_putative_matches.empty())
         {
-          map_PutativesMatches.insert( make_pair( make_pair(I,J), std::move(vec_putative_matches) ));
+          map_PutativesMatches[std::make_pair(I,J)].emplace(descType, std::move(vec_putative_matches));
```

Comment:
maybe to be better safe than sorry
assert(map_PutativesMatches.count(std::make_pair(I,J)) == 0)